grafana-formula/grafana/init.sls

```diff
@@ -95,6 +95,21 @@
   file.absent
 {%- endif %}
 
+# HA and SAP dashboards
+# * HA:
+{%- if salt['pillar.get']('grafana:dashboards:add_ha_dashboards', False) %}
+  pkg.installed:
+    - name:
+      - grafana-ha-cluster-dashboards
+{%- endif %}
+# * SAP:
+{%- if salt['pillar.get']('grafana:dashboards:add_sap_dashboards', False) %}
+  pkg.installed:
+    - pkgs:
+      - grafana-sap-hana-dashboards
+      - grafana-sap-netweaver-dashboards
```

Comment:
I think the netweaver dashboard should go in a separate setting, we don't usually install them both in the same node.

Reply:
ok fair point. thx